src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import { shell } from '../../../shell';
+
+export async function startKubernetes() {
+    shell.exec('minikube start').then(() => {
```

Comment:
doesn't need to happen here, but we should file an issue to: 
- confirm if minikube is installed
- prompt the user if we can install it on behalf of them

Reply:
will file once this merges.

now checks return code.